typescript/cli/src/validator/address.ts

```diff
@@ -0,0 +1,139 @@
+import { GetPublicKeyCommand, KMSClient } from '@aws-sdk/client-kms';
+import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+// @ts-ignore
+import asn1 from 'asn1.js';
+import { ethers } from 'ethers';
+
+import { CommandContext } from '../context/types.js';
+import { log, logBlue } from '../logger.js';
+
+export async function getValidatorAddress({
+  context,
+  bucket,
+  keyId,
+}: {
+  context: CommandContext;
+  bucket?: string;
+  keyId: string;
+}) {
+  if (!bucket && !keyId) {
+    throw new Error('Must provide either an S3 bucket or a KMS Key ID.');
+  }
+
+  // User must have below env variables configured beforehand
+  const accessKeyId = process.env.AWS_ACCESS_KEY_ID;
+  const secretAccessKey = process.env.AWS_SECRET_ACCESS_KEY;
+  const region = process.env.AWS_REGION;
```

Comment:
Please handle these required args like we handle keys here:
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/cli/src/commands/options.ts#L48

Meaning you should:
1. Allow users to set them via command args
2. Add these fields to `envScheme` 
3. Default to the env var when the arg is not set
4. Prompt the user for any required values that weren't provided